visualization/settings/templates.py

```diff
@@ -0,0 +1,59 @@
+import os
+from pathlib import Path
+
+path = os.path.abspath(Path(__file__).parent.parent.parent)
+header = f"export PYTHONPATH='{path}:$PYTHONPATH'"
```

Comment:
I think you need to package installed anyway, so maybe remove python path and just ask to follow instructions in prerequisites docs for local installation?